workflows/epigenetics/ChIPseq/README.md

```diff
@@ -0,0 +1,20 @@
+# ChIP Workflow
+
+## Inputs
+- The workflow needs a single input which is a list of dataset pairs of fastqsanger.
+
+## Processing
+- The workflow will remove illumina adapters and low quality bases and filter out any read smaller than 15bp
+- The filtered reads are mapped with bowtie2 with default parameters
+- The BAM is filtered to keep only MAPQ30 and concordant pairs
+- The peaks are called with macs2 which at the same time generate a coverage.
```

Comment:
```suggestion
- The peaks are called with macs2 which at the same time generates a coverage file
```
Which file format is this ? Might be nice to say
```
The peaks are called with macs2 which at the same time generates a coverage file in bedgraph format.
```
(no idea which format this is, just guessing here ...)